### ISSUE TITLE
Not able to compile.

### ISSUE THREAD

**Issue by DeepaAC** [java8/Java8InAction#36]
Not able to compile the code as its throwing lots of ERROR.

**Comment by peiyaqi** [comment]
me too

**Comment by MatrixMike** [comment]
Yes I tried too ages ago. I think repos that are followed should be regularly flagged to be checked. At least we could more specifically detail the requirements of the repo. This is a common problem for me so I lend my support to any system that promotes this.
